src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -140,9 +138,16 @@ def generate_initial_population(self, pop_data: SimulantData) -> None:
             np.round(pop["age"] * 365.25), unit="days"
         )
 
-        # Add Social Security Numbers
-        pop["ssn"] = self.ssn_generator.generate(pop).ssn
-        pop["ssn"] = self.ssn_generator.remove_ssn(pop["ssn"], self.proportion_with_no_ssn)
+        # Deterimne if simulants have SSN
+        pop["ssn"] = False
+        # Give native born simulants a SSN  
+        non_natives_idx = pop.loc[~pop["born_in_us"]].index  # Get index for sims born outside of United States
```

Comment:
This feels like a double negative. Can we not just use `pop.loc[pop["born_in_us"], "ssn"] = True` in the next line, and define this variable just before `ssn_idx`?